app-emulation/winetricks/winetricks-20170614.ebuild

```diff
@@ -21,7 +21,7 @@ SRC_URI="${SRC_URI}
 	kde? ( https://dev.gentoo.org/~tetromino/distfiles/wine/${wtg}.tar.bz2 )"
 
 DESCRIPTION="Easy way to install DLLs needed to work around problems in Wine"
-HOMEPAGE="http://winetricks.org http://wiki.winehq.org/winetricks"
```

Comment:
Use https://wiki.winehq.org/Winetricks and/or https://github.com/Winetricks/winetricks, please